### PR TITLE
En 974 system labels

### PR DESCRIPTION
**Pull request by DMinnai**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: src/ui/common/form/RenderTextAreaInput.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+RenderTextAreaInput.propTypes = {
+  input: PropTypes.shape({}),
+  name: PropTypes.string,
+  label: PropTypes.node,
+  placeholder: PropTypes.string,
+  meta: PropTypes.shape({}),
+  help: PropTypes.node,
+  disabled: PropTypes.bool,
+  type: PropTypes.string,
```
Comment (fedelocci) [inline src/ui/common/form/RenderTextAreaInput.js]: remove it's a textarea

File: src/ui/common/form/RenderTextAreaInput.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+RenderTextAreaInput.defaultProps = {
+  input: {},
+  name: '',
+  label: '',
+  placeholder: '',
+  meta: {},
+  help: null,
+  disabled: false,
+  type: 'text',
```
Comment (fedelocci) [inline src/ui/common/form/RenderTextAreaInput.js]: remove type:'text'

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

File: test/ui/common/form/RenderTextAreaInput.test.js [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+const META = { touched: true, error: '' };
+const META_WITH_ERROR = { touched: true, error: 'Required' };
+
+describe('RenderTextInput', () => {
+  let textInput;
+
+  it('render component without crash', () => {
+    const element = RenderTextAreaInput({ input: INPUT, label: LABEL, meta: META });
+    textInput = shallow(element);
+    expect(textInput.exists()).toEqual(true);
```
Comment (fedelocci) [inline test/ui/common/form/RenderTextAreaInput.test.js]: it's prefer use toBe(true)

File: test/ui/common/form/RenderTextAreaInput.test.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+  });
+
+  it('render error message for textInput', () => {
+    const element = RenderTextAreaInput({ input: INPUT, label: LABEL, meta: META_WITH_ERROR });
+    textInput = shallow(element);
+    const textInputHelpBlock = textInput.find('.help-block').first();
+    expect(textInputHelpBlock.exists()).toEqual(true);
```
Comment (fedelocci) [inline test/ui/common/form/RenderTextAreaInput.test.js]: toBe

File: test/ui/labels/add/AddLabelsPage.test.js [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+import { shallow } from 'enzyme';
+import AddLabelsPage from 'ui/labels/add/AddLabelsPage';
+
+describe('AddUserPage', () => {
+  let component;
+  beforeEach(() => {
+    component = shallow(<AddLabelsPage />);
+  });
+  it('renders without crashing', () => {
+    expect(component.exists()).toEqual(true);
```
Comment (fedelocci) [inline test/ui/labels/add/AddLabelsPage.test.js]: toBe

File: test/ui/labels/add/AddLabelsPage.test.js [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+  });
+
+  it('verify if exist InternalPage with class AddUserPage', () => {
+    expect(component.find('InternalPage').hasClass('AddLabelsPage')).toEqual(true);
```
Comment (fedelocci) [inline test/ui/labels/add/AddLabelsPage.test.js]: toBe

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Comment by entando-service** [comment]
All is well

**Pull request merged**